src/components/GeneralInput.tsx

```diff
@@ -0,0 +1,25 @@
+import { CloudMoonRain } from 'lucide-react';
+import React from 'react';
+import { Input } from './lib/ui/input';
+import { Trykker } from 'next/font/google';
+
+type inputProps = {
+    primaryTextVisible: boolean,
+    secondaryTextVisible: boolean,
+    primaryText: string,
```

Comment:
```suggestion
    primaryText?: string,
```
Adding these as optional avoids the need to add a boolean flag

Reply:
Should be fixed now!